hs_composite_resource/models.py

```diff
@@ -586,11 +586,8 @@ def get_data_services_urls(self):
                     ] for data_type in resource_data_types
                 ):
                     wms_url = (
-                        f'{settings.HSWS_GEOSERVER_URL}/wms?'
```

Comment:
Let's go ahead and simplify this whole function while we're in here.  The URL is formatted the same for all 3, it just uses wms, wfs, or wcs.  All of these if/else blocks are just confusing.  Below should be functionally the same.
```
    def get_data_services_urls(self):
        """
        Generates data services URLs for the resource.

        If the resource contains any GeoFeature or GeoRaster content, and if it's public,
        generate data service endpoints.
        """

        wfs_url = None
        wms_url = None
        wcs_url = None
        if self.raccess.public:
            try:
                resource_data_types = [lf.data_type for lf in self.logical_files]
                service_url = (
                        f'{settings.HSWS_GEOSERVER_URL}/HS-{self.short_id}/' +
                        '{}?request=GetCapabilities'
                    )
                if 'GeographicFeature' in resource_data_types:
                    wfs_url = service_url.format('wfs')
                    wms_url = service_url.format('wms')
                if 'GeographicRaster' in resource_data_types:
                    wcs_url = service_url.format('wcs')
                    wms_url = service_url.format('wms')

            except Exception as e:
                logger.exception("get_data_services_urls: " + str(e))

        data_services_urls = {
            'wms_url': wms_url,
            'wfs_url': wfs_url,
            'wcs_url': wcs_url
        }

        return data_services_urls
```